dashboards-observability/public/components/event_analytics/explorer/save_panel/__tests__/__snapshots__/save_panel.test.tsx.snap

```diff
@@ -785,5 +787,198 @@ exports[`Saved query table component Renders saved query table 1`] = `
       </div>
     </div>
   </EuiFormRow>
+  <EuiSpacer
+    size="s"
+  >
+    <div
+      className="euiSpacer euiSpacer--s"
+    />
+  </EuiSpacer>
+  <EuiTitle
+    size="xxs"
+  >
+    <h3
+      className="euiTitle euiTitle--xxsmall"
+    >
+      Units of Measure
+    </h3>
+  </EuiTitle>
+  <EuiFormRow
+    describedByIds={Array []}
+    display="row"
+    fullWidth={false}
+    hasChildLabel={true}
+    hasEmptyLabelSpace={false}
+    labelType="label"
+  >
+    <div
+      className="euiFormRow"
+      id="random_html_id-row"
+    >
+      <div
+        className="euiFormRow__fieldWrapper"
+      >
+        <EuiSelect
+          data-test-subj="eventExplorer__metricMeasureSaveSelectBox"
+          id="random_html_id"
+          onBlur={[Function]}
+          onChange={[Function]}
+          onFocus={[Function]}
+          options={
+            Array [
+              Object {
+                "text": "seconds (s)",
+                "value": "seconds (s)",
+              },
+              Object {
+                "text": "hours (h)",
+                "value": "hours (h)",
+              },
+              Object {
+                "text": "celsius (C)",
+                "value": "celsius (C)",
+              },
+              Object {
+                "text": "farenheit (F)",
+                "value": "farenheit (F)",
+              },
+              Object {
+                "text": "meters (m)",
+                "value": "meters (m)",
+              },
+              Object {
+                "text": "kilometers (k)",
+                "value": "kilometers (k)",
+              },
```

Comment:
Can you please update this to be generated in runtime. Like done here: https://github.com/opensearch-project/observability/blob/e88ab448e6d3f30032ff5d48189ba0d25bbebf2d/dashboards-observability/public/components/metrics/top_menu/metrics_export_panel.tsx#L135